static/templates/settings/realm-filter-settings-admin.handlebars

```diff
@@ -0,0 +1,35 @@
+<div id="filter-settings" class="settings-section">
+  <div class="settings-section-title"><i class="icon-vector-filter settings-section-icon"></i>Custom realm filters</div>
+  <div class="admin-table-wrapper">
```

Comment:
This should add a description explaining how "realm filters" work -- i.e. that it is a set of regular expressions that will be automatically linkified when used in Zulip message bodies or topics.